src/clickChart.js

```diff
@@ -0,0 +1,74 @@
+import Chart from 'chart.js/auto';
+import { pageData } from './apiCalls';
+import { sortByHits } from './recipes';
+
+const makeRecipeClickChart = () => {
+    const chartContainer = document.querySelector('#chartContainer');
+    chartContainer.innerHTML = `<canvas id="clickChart"></canvas>`;
+    const sortedRecipes = sortByHits(pageData.allRecipes).slice(0, 9);
+    new Chart(document.querySelector('#clickChart'),
+        {
+            type: 'bar',
+            data: {
+                labels: sortedRecipes.map(recipe => splitLabelLength(recipe.name)),
+                fontColor: 'white',
+                datasets: [
+                    {
+                        label: 'Hits',
+                        data: sortedRecipes.map(recipe => recipe.hits),
+                        borderColor: 'red',
+                        backgroundColor: 'red',
```

Comment:
is it possible for us to use a color that goes with the current palette, like `#fbbf83` or `#faac5f`?

Reply:
done!  Looks wayyyy better